Faucet docs don't build against current stable version of Sphinx (1.7.0)

I followed the README.rst in the docs directory, when you use `pip3 install -r requirements.txt`, this will pull in sphinx 1.7.0. 

running `make html` produces the following error: 

> Running Sphinx v1.7.0
> loading pickled environment... not yet created
> usage: usage: sphinx-build [OPTIONS] -o <OUTPUT_PATH> <MODULE_PATH> [EXCLUDE_PATTERN, ...]
> sphinx-build: error: unrecognized arguments: ../faucet
> Makefile:20: recipe for target 'html' failed
> make: *** [html] Error 2

This error is caused by the following breaking changes to sphinx:
#3668: The arguments has changed of main functions for each command

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -170,7 +170,7 @@ def run_apidoc(_):
     """Call sphinx-apidoc on faucet module"""
     from sphinx.apidoc import main as apidoc_main
     cur_dir = os.path.abspath(os.path.dirname(__file__))
-    apidoc_main([None, '-e', '-o', 'source/apidoc', '../faucet'])
+    apidoc_main(['-e', '-o', 'source/apidoc', '../faucet'])
 
 def setup(app):
     """Over-ride Sphinx setup to trigger sphinx-apidoc."""

--- a/docs/requirements.txt
+++ b/docs/requirements.txt
@@ -1,3 +1,3 @@
 -r ../requirements.txt
-sphinx>=1.6
-sphinx_rtd_theme
+sphinx==1.7.0
+sphinx_rtd_theme==0.2.4

